### DISCUSSION TITLE
lighthouse is running only on mobile view mode as default, how to change to desktop mode?

### DISCUSSION THREAD

**Discussion by ShobanMano** [GoogleChrome/lighthouse#12660] [Q&A]
**Problem Statement:**

I am using lighthouse and puppeteer for my "Web performance testing" , I could not run my scripts against "Desktop Mode" , as it always running against "Mobile Mode" even browser window size is set to full screen. 

_Is there any options or flag to enable for desktop mode in code?_

**Tech details:**

```
lighthouse - 8.0.0
puppeteer - 9.1.1
```

**Code Snippet:**

```
before(async() => {
       browser = await puppeteer.launch({
            headless: false,
            args: [`--remote-debugging-port=${8042}`,'--no-sandbox', '--disable-setuid-sandbox', '--start-fullscreen']
        });
        lighthouseResult = await lighthouse(APP_URL, {
        port: 8042,
        disableStorageReset: true,
       });
    });
```

**Comment by patrickhulce** [comment]
Yep! Use the [desktop config](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/desktop-config.js).

```js
await lighthouse(
  APP_URL, 
  {port: 8042, disableStorageReset: true}, 
  require('lighthouse/lighthouse-core/config/desktop-config.js')
)
```

**Reply by ShobanMano** [reply]
Hello @patrickhulce 

it is working as you suggested, Thanks so much :)
